src/patternfly/components/ClipboardCopy/examples/clipboard-copy-expansion-example.hbs

```diff
@@ -0,0 +1,24 @@
+{{#> clipboard-copy}}
+	{{#> clipboard-copy-group}}
+    {{#> clipboard-copy-group-toggle clipboard-copy-group-toggle--attribute='id="expand-button-1" aria-labelledby="expand-button-1 text-input-3" aria-label="Show More" aria-expanded="false" aria-controls="ex-content-1"'}}{{/clipboard-copy-group-toggle}}
+    {{#> form-control controlType="input" input="true" form-control--attribute='type="text" value="Got a lot of text here, need to see all of it? Click that arrow on the left side and check out the resulting expansion." id="text-input-3" aria-label="copyable input"'}}
+    {{/form-control}}
+	  {{#> clipboard-copy-group-copy clipboard-copy-group-copy--attribute='id="copy-button-3" aria-labelledby="copy-button-3 text-input-3"'}}{{/clipboard-copy-group-copy}}
+	{{/clipboard-copy-group}}
+  {{#> clipboard-copy-expanded-content clipboard-copy-expanded-content--attribute='id="ex-content-1"'}}
+    Got a lot of text here, need to see all of it? Click that arrow on the left side and check out the resulting expansion.
+  {{/clipboard-copy-expanded-content}}
+{{/clipboard-copy}}
+<br>
+<br>
+{{#> clipboard-copy clipboard-copy--IsExpanded="true"}}
+	{{#> clipboard-copy-group}}
+    {{#> clipboard-copy-group-toggle clipboard-copy-group-toggle--attribute='id="expand-button-2" aria-labelledby="expand-button-2 text-input-4" aria-label="Show More" aria-expanded="false" aria-controls="ex-content-2"'}}{{/clipboard-copy-group-toggle}}
```

Comment:
`aria-expanded` should be true, but I think you can generate all of these attributes dynamically in the `.hbs` files by referencing `clipboard-copy--IsExpanded` for the `aria-label` and `aria-expanded` values, and use an `id` on `{{#> clipboard-copy}}` to generate the `id`, `aria-labelledby` and `aria-controls` values for `clipboard-copy-group-toggle`,  `clipboard-copy-group-copy`, and `clipboard-copy-expanded-content`

Reply:
Oh yep, made those updates :)